lib/generators/utils/entry.js

```diff
@@ -55,7 +55,7 @@ module.exports = (self, answer) => {
 					self.prompt([
 						InputValidate(
 							`${entryProp}`,
-							`What is the location of "${entryProp}"? [example: "./src/${entryProp}"]`,
+							`What is the location of "${entryProp}"? [example: './src/${entryProp}']`,
```

Comment:
This one was showing example in double quotes when other lines use single quotes, changed it. If there was no good reason for it?